packages/calcite-components/src/components/input-text/input-text.tsx

```diff
@@ -415,14 +415,15 @@ export class InputText
   //--------------------------------------------------------------------------
 
   keyDownHandler = (event: KeyboardEvent): void => {
-    if (this.readOnly || this.disabled) {
+    if (this.readOnly || this.disabled || event.defaultPrevented) {
       return;
     }
+
     if (this.isClearable && event.key === "Escape") {
       this.clearInputTextValue(event);
       event.preventDefault();
     }
-    if (event.key === "Enter" && !event.defaultPrevented) {
+    if (event.key === "Enter") {
       if (submitForm(this)) {
```

Comment:
For the input text, we are only calling preventDefault if the input is inside a form. I wonder if we should call `event.preventDefault()` for the enter key all the time? cc @jcfranco. Is it inconsistent to only call this when inside a form?

Reply:
> Is it inconsistent to only call this when inside a form?

Not according to [our conventions](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/README.md#native-event-cancellation). We should only be canceling native events if the component reacts to an event for a specific interaction.